<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>645</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>684</width>
    <height>645</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>684</width>
    <height>645</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background: qlineargradient(
    x1:0, y1:0, x2:0, y2:1,
    stop:0 #F66DBB,
    stop:1 #CC63CA
);
}</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 10px; /* скругление углов */
    padding: 6px 12px;
    color: white;
     background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #CD0074,  /* цвет без градиента сверху */
        stop:0.5 #CD0074,
        stop:1 #A325A4   /* градиентный цвет снизу */
	);
}

QPushButton:pressed {
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #9A0058,  /* более тёмный цвет сверху */
        stop:0.5 #9A0058,
        stop:1 #6E1E73   /* более тёмный цвет снизу */
    );
}

QPushButton:hover {
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #9B005C,  /* более насыщенный, тёмный цвет сверху */
        stop:0.5 #9B005C,
        stop:1 #7A1E7B   /* более насыщенный, тёмный цвет снизу */
    );
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="timeSlider">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>391</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider{
background: transparent;
}
QSlider::groove:horizontal {
    height: 7px;
    background: #CD0074;               /* цвет оставшейся части */
    border-radius: 3.5px;           /* половина высоты для скругления */
}

QSlider::sub-page:horizontal {
    background: #CD0074;            /* цвет пройденной части */
    border-radius: 3.5px;
}

QSlider::add-page:horizontal {
    background: #FFE9F6;            /* цвет оставшейся части (альтернативный) */
    border-radius: 3.5px;
}

QSlider::handle:horizontal {
    background: #FFE9F6;            /* цвет ползунка */
    width: 20px;
    height: 20px;
    margin: -4px 0;                 /* чтобы ползунок не обрезался */
    border-radius: 7px;             /* круглый ползунок */
}
</string>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="next_btn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 10px; /* скругление углов */
    padding: 6px 12px;
    color: white;
     background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #CD0074,  /* цвет без градиента сверху */
        stop:0.5 #CD0074,
        stop:1 #A325A4   /* градиентный цвет снизу */
	);
}

QPushButton:pressed {
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #9A0058,  /* более тёмный цвет сверху */
        stop:0.5 #9A0058,
        stop:1 #6E1E73   /* более тёмный цвет снизу */
    );
}

QPushButton:hover {
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #9B005C,  /* более насыщенный, тёмный цвет сверху */
        stop:0.5 #9B005C,
        stop:1 #7A1E7B   /* более насыщенный, тёмный цвет снизу */
    );
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="previous_btn">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>49</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 10px; /* скругление углов */
    padding: 6px 12px;
    color: white;
     background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #CD0074,  /* цвет без градиента сверху */
        stop:0.5 #CD0074,
        stop:1 #A325A4   /* градиентный цвет снизу */
	);
}

QPushButton:pressed {
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #9A0058,  /* более тёмный цвет сверху */
        stop:0.5 #9A0058,
        stop:1 #6E1E73   /* более тёмный цвет снизу */
    );
}

QPushButton:hover {
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #9B005C,  /* более насыщенный, тёмный цвет сверху */
        stop:0.5 #9B005C,
        stop:1 #7A1E7B   /* более насыщенный, тёмный цвет снизу */
    );
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="repeat_btn">
   <property name="geometry">
    <rect>
     <x>191</x>
     <y>50</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 10px;
    padding: 6px 12px;
    color: white;
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #CD0074,
        stop:0.5 #CD0074,
        stop:1 #A325A4
    );
    border: none;
}

QPushButton:pressed {
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #9A0058,
        stop:0.5 #9A0058,
        stop:1 #6E1E73
    );
}

QPushButton:hover {
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #9B005C,
        stop:0.5 #9B005C,
        stop:1 #7A1E7B
    );
}

QPushButton:focus {
    outline: none;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="currentTime_label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background: transparent;
}</string>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="time_of_track">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>100</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background: transparent;
}</string>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background: transparent;
}</string>
   </property>
   <property name="text">
    <string>Текущий трек:</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_track_label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background: transparent;
}</string>
   </property>
   <property name="text">
    <string>Не играет</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>651</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
    border-radius: 15px;           /* Скругление углов радиусом 15px */
    background-color: #FFE9F6;     /* Заливка сплошным цветом */
}
</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>631</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>629</width>
          <height>449</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <widget class="QListWidget" name="listWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>631</width>
           <height>551</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
